### PR TITLE
Feature/design change

### PR DESCRIPTION
**Pull request by SeunghunSunmoonLee**

### CODE REVIEW

File: src/components/Main/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Button, ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem } from 'reactstrap'
 import './index.css'
-
+import tokenSales from '../TokenSale/tokenSales.js'
```
Comment (geastwood) [inline src/components/Main/index.js]: No need of using extension .js, if you run `yarn lint:js`, the linter will hint you as well.

**Comment by geastwood** [comment]
This PR contains commit from TokenSale PR?

**Comment by geastwood** [comment]
Seems there is a problem after clicking the link.

screen shot 2018-06-01 at 20 49 53

File: src/components/GalleryTeaser/index.js [hunk 2]
```diff
@@ -13,32 +13,13 @@ const GallaryTeaser = () => (
...
-            </a>&nbsp;or&nbsp;
-            <a
-              href="https://www.youtube.com/channel/UCV_O9CKVKr4-KJVC-v8FcVw"
-              rel="noopener noreferrer"
-              target="_blank"
-              style={{ color: '#ce38da' }}
-            >
-              Youtube
-            </a>&nbsp;to get latest news!
+            Ares is Scalable Blockchain Social Gaming Network.
```
Comment (chutium) [inline src/components/GalleryTeaser/index.js]: Please revert, we changed the subtitle here, zeplin was outdated.

File: src/components/Main/index.js [hunk 3]
```diff
@@ -44,12 +44,12 @@ class Downloads extends React.Component {
 }
 
 const Main = () => (
-  <div className="container p-3 p-md-5 main-content">
+  <div className="container p-3 p-md-5 main-content" style={{paddingTop: '5em !important'}}>
     <div className="main-shape-2" />
-    <h1 className="display-2 headline" style={{ color: '#fff' }}>
-      Blockchain. <br /> Gaming. <br />Network.
+    <h1 className="display-2 headline" style={{ paddingBottom: '10px', color: '#fff' }}>
+      Scalable Blockchain <br /> Social Gaming Network
```
Comment (chutium) [inline src/components/Main/index.js]: Please revert, we changed the headline text here, zeplin was outdated......

File: src/components/Main/index.js [hunk 4]
```diff
@@ -44,12 +44,12 @@ class Downloads extends React.Component {
...
     </h1>
-    <p className="lead sub-headline">Scalable Blockchain All-in-One Solution for Social Game Creators</p>
+    <p className="lead sub-headline">All-in-One Solution for Social Game Creators</p>
```
Comment (chutium) [inline src/components/Main/index.js]: Please revert, we changed the headline text here, zeplin was outdated......

File: src/components/Main/index.js [hunk 5]
```diff
@@ -65,11 +65,39 @@ const Main = () => (
         <a href="https://t.me/ares_tech" rel="noopener noreferrer" target="_blank" style={{ color: '#ce38da' }}>
           <Button className="main__button" block>
             <i className="fab fa-telegram-plane" />
-            <span style={{ paddingLeft: 10 }}>Join our Telegram group</span>
+            <span style={{ paddingLeft: 10 }}>Chat with us</span>
```
Comment (chutium) [inline src/components/Main/index.js]: Please revert, we changed the headline text here, zeplin was outdated......

**Comment by chutium** [comment]
Please revert the changes, we changed some text, zeplin was outdated......

We'd better close this PR for now.

File: src/components/GalleryTeaser/index.js [hunk 6]
```diff
@@ -13,32 +13,13 @@ const GallaryTeaser = () => (
...
           </p>
         </Col>
       </Row>
       {chunk(pictures, 3).map((row, index) => (
         // eslint-disable-next-line
-        <Row key={index}>
+        <Row key={index} >
```
[the remark this answers is not in the document]
Comment (SeunghunSunmoonLee) [reply src/components/GalleryTeaser/index.js]: ok I'll implement prettier if you want? no-semi or semi option?

File: src/components/Main/index.js (hunk 1 above)
Comment (SeunghunSunmoonLee) [reply src/components/Main/index.js]: gotcha np

File: src/components/GalleryTeaser/index.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (geastwood) [reply src/components/GalleryTeaser/index.js]: there is a .prettierrc file at root, you editor will be able to recognise the config file.

**Pull request closed without merging**